src/amo/components/AddonsCard/index.js

```diff
@@ -19,6 +19,7 @@ type Props = {|
   className?: string,
   editing?: boolean,
   loading?: boolean,
+  placeholderText: string | null,
```

Comment:
I'm not sure it makes sense to add this to the `AddonsCard` at all. I think it makes more sense to just add it to the `Collection` and `CollectionList`.